packages/server/src/config.ts

```diff
@@ -2,6 +2,8 @@ import {resolve} from 'path'
 import {ciLog} from './ciLog'
 import {resolveBinAsync} from './resolve-bin-async'
 import {synchronizeFiles} from './synchronizer'
+import _ from 'lodash'
```

Comment:
This is unused. One other reason why we shouldn't use the `_` is that typescript considers `_ `as a possibly unused value and won't complain when we don't use it.

Reply:
Happy to merge if we can clean this up. I will leave it up to you for whether or not you want to import deeply.